src/php/ext/grpc/channel.c

```diff
@@ -296,12 +406,11 @@ PHP_METHOD(Channel, __construct) {
     strcat(key, creds->hashstr);
   }
   channel->wrapper = malloc(sizeof(grpc_channel_wrapper));
+  channel->wrapper->ref_count = 0;
   channel->wrapper->key = key;
   channel->wrapper->target = strdup(target);
   channel->wrapper->args_hashstr = strdup(sha1str);
   channel->wrapper->creds_hashstr = NULL;
-  channel->wrapper->ref_count = 1;
-  channel->wrapper->is_valid = true;
   if (creds != NULL && creds->hashstr != NULL) {
     php_grpc_int creds_hashstr_len = strlen(creds->hashstr);
```

Comment:
In this case, the channel comes with 0 refcount?

Reply:
Yes. This is just the initial status.
I put `ref_count += `1 inside [`create_channel()`](https://github.com/grpc/grpc/pull/15218/files/591fd4657130e335e0ae91747b2190498752a3e7#diff-f942bdda09aac4a837cef783506850f0R248) function.